tests/libtest.sh

```diff
@@ -21,8 +21,8 @@ lv_exists() {
 }
 
 clean_config_files() {
-  rm -f /etc/sysconfig/docker-storage-setup
-  rm -f /etc/sysconfig/docker-storage
+  rm -f $1
+  rm -f $2
```

Comment:
Not in the patches I just pushed.  All changes for infile and outfile lower case.